app/Decsys/Repositories/LiteDb/LiteDbWordlistRepository.cs

```diff
@@ -0,0 +1,22 @@
+using AutoMapper;
+using Decsys.Constants;
+using Decsys.Data;
+using Decsys.Models.Wordlist;
+using Decsys.Repositories.Contracts;
+using LiteDB;
+
+
+namespace Decsys.Repositories.LiteDb;
+
+public class LiteDbWordlistRepository : IWordlistRepository
+{
+    public UserWordlist List(string ownerId) 
+    {
+        throw new NotImplementedException();
```

Comment:
Worth adding a comment as to why these aren't implemented. At the top of the class if it's going to be every method. 
I'm guessing it's as word list management won't be in the workshop mode? 
Otherwise if it's just implemented in a future PR take this comment as it being written down somewhere and no need to add TODOs